fabric-item-api-v1/src/main/java/net/fabricmc/fabric/api/item/v1/FabricItem.java

```diff
@@ -122,4 +124,15 @@ default boolean isSuitableFor(ItemStack stack, BlockState state) {
 	default ItemStack getRecipeRemainder(ItemStack stack) {
 		return ((Item) this).hasRecipeRemainder() ? ((Item) this).getRecipeRemainder().getDefaultStack() : ItemStack.EMPTY;
 	}
+
+	/**
+	 * This is a stack-aware version of {@link Item#getFoodComponent()}.
+	 * Note that simple food component can also be set via {@link Item.Settings#food(FoodComponent)}.
+	 * If you want to get a food component for a stack, is <b>recommended</b> to use the stack version of this method: {@link FabricItemStack#getFoodComponent()}.
```

Comment:
```suggestion
	 * If you want to get a food component for a stack, is <strong>recommended</strong> to use the stack version of this method: {@link FabricItemStack#getFoodComponent()}.
```
Using `<strong>` for this use case is a [common HTML recommendation](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b).